package/MDAnalysis/visualization/streamlines.py

```diff
@@ -85,7 +86,7 @@ def produce_grid(tuple_of_limits, grid_spacing):
     return grid
 
 
-def split_grid(grid, num_cores):
+def split_grid(grid: NDArray, num_cores: int) -> Tuple[List[List[int]], List[List[int]], int, int]:
```

Comment:
Isn't the return type a `List` instead of `Tuple` here?

Reply:
Yes I made a mistake out there